src/backend/core/api/viewsets.py

```diff
@@ -686,15 +737,35 @@ class TemplateViewSet(
 ):
     """Template ViewSet"""
 
+    filter_backends = [filters.OrderingFilter]
     permission_classes = [
         permissions.IsAuthenticatedOrSafe,
         permissions.AccessPermission,
     ]
+    ordering = ["-created_at"]
+    ordering_fields = ["created_at", "updated_at", "title"]
     serializer_class = serializers.TemplateSerializer
-    access_model_class = models.TemplateAccess
-    resource_field_name = "template"
     queryset = models.Template.objects.all()
 
+    def get_queryset(self):
```

Comment:
I guess you need to clean the ˋlist` action because you filter objects here already, no?

Reply:
I didn't find any optimization to make. Filtering in get_queryset is what should apply to any actions and the filtering here is what should apply only to the `list` action.